projects/1.16/assets/fantasys-furniture/fantasyfurniture/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+  "block.fantasyfurniture.decorations.venthyr.chalices": "温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.chalices.stacked": "堆积的温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.food_0": "温西尔食品 1",
+  "block.fantasyfurniture.decorations.venthyr.food_1": "温西尔食品 2",
+  "block.fantasyfurniture.decorations.venthyr.platter": "温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.platter.stacked": "堆积的温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups": "温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups.stacked": "堆积的温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_set": "温西尔茶具",
+  "block.fantasyfurniture.decorations.venthyr.tomes": "温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.tomes.stacked": "堆积的温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.widow_bloom": "寡妇花瓶",
+  "block.fantasyfurniture.dunmer.bed_double": "邓默双人床",
+  "block.fantasyfurniture.dunmer.bed_double.occupied": "这张床已被占用",
+  "block.fantasyfurniture.dunmer.bed_single": "邓默单人床",
+  "block.fantasyfurniture.dunmer.bed_single.occupied": "这张床已被占用",
+  "block.fantasyfurniture.dunmer.bench": "邓默长凳",
+  "block.fantasyfurniture.dunmer.bench.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.bookshelf": "邓默书架",
+  "block.fantasyfurniture.dunmer.carpet": "邓默地毯",
+  "block.fantasyfurniture.dunmer.chair": "邓默椅",
+  "block.fantasyfurniture.dunmer.chair.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.chandelier": "邓默枝形吊灯",
+  "block.fantasyfurniture.dunmer.chest": "邓默宝箱",
+  "block.fantasyfurniture.dunmer.cushion": "邓默软垫",
+  "block.fantasyfurniture.dunmer.cushion.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.desk_left": "邓默办公桌左",
+  "block.fantasyfurniture.dunmer.desk_right": "邓默办公桌右",
+  "block.fantasyfurniture.dunmer.door_double": "邓默门 2",
+  "block.fantasyfurniture.dunmer.door_single": "邓默门 1",
+  "block.fantasyfurniture.dunmer.drawer": "邓默抽屉",
+  "block.fantasyfurniture.dunmer.dresser": "邓默德莱瑟",
+  "block.fantasyfurniture.dunmer.floor_light": "邓默落地灯",
+  "block.fantasyfurniture.dunmer.painting_small": "邓默绘画小",
+  "block.fantasyfurniture.dunmer.painting_wide": "邓默宽幅绘画",
+  "block.fantasyfurniture.dunmer.shelf": "邓默货架",
+  "block.fantasyfurniture.dunmer.sofa": "邓默沙发",
+  "block.fantasyfurniture.dunmer.sofa.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.stool": "邓默凳子",
+  "block.fantasyfurniture.dunmer.stool.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.table_large": "邓默大桌子",
+  "block.fantasyfurniture.dunmer.table_small": "邓默小桌",
+  "block.fantasyfurniture.dunmer.table_wide": "邓默宽桌",
+  "block.fantasyfurniture.dunmer.wall_light": "邓默壁灯",
+  "block.fantasyfurniture.dunmer.wardrobe_bottom": "邓默衣柜",
+  "block.fantasyfurniture.dunmer.wardrobe_top": "邓默衣柜上衣",
+  "block.fantasyfurniture.dunmer.wool": "邓默羊毛",
+  "block.fantasyfurniture.furniture_station": "家具站",
+  "block.fantasyfurniture.nordic.bed_double": "北欧双人床",
```

```suggestion
  "block.fantasyfurniture.nordic.bed_double": "诺德双人床",
```
以更加符合本模组特点，此处对Nordic的翻译应采取《上古卷轴》系列中对Nord的翻译而译为“诺德”，而非现实中的北欧，此问题请全文通查